client/app/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import { AuthProvider } from "./context/AuthContext";
+import { useAuth } from "./context/AuthContext";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import Home from "./screens/home";
+import Login from "./screens/login";
+import { Button } from "react-native";
+import Register from "./screens/register";
+
+const Stack = createNativeStackNavigator();
+
+const App = () => {
+  return (
+    <AuthProvider>
+      <Layout></Layout>
+    </AuthProvider>
+  );
+};
+
+export default App;
+
+export const Layout = () => {
+  const { authState, onLogout } = useAuth();
+  const [state, setState] = useState(0);
```

Comment:
could it be somethin
```suggestion
  const [isRegistered, setIsRegistered] = useState(false);
```